sel4-extract/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# set minimum cmake verision
+cmake_minimum_required(VERSION 3.0.0)
+
+# set project name
+project(sel4-extract)
+
+# add executable
+add_executable (sel4-extract sel4-extract.c)
+
+# set compiler options
+set(FLAGS_WAR_AS_ERR -Wall -Wpedantic -Werror)
+# set(FLAGS_MEM_DBG -fsanitize=address -fsanitize=leak -fsanitize=bounds)
```

Comment:
I recommend using a configuration flag (i.e. what you pass to `cmake -DSOMETHING=1`) instead of comment and conditionally add either directly to target_compile_options or to a variable.

Reply:
Fixed.